py/desispec/fiberbitmasking.py

```diff
@@ -0,0 +1,146 @@
+"""
+desispec.fiberbitmasking
+==============
+
+Functions to properly take FIBERSTATUS into account in the variances for data reduction
+"""
+
+from __future__ import absolute_import, division
+import numpy as np
+from astropy.table import Table
+
+from desiutil.log import get_logger
+from desispec.maskbits import fibermask as fmsk
+from desispec.maskbits import specmask
+
+
+def get_fiberbitmasked_frame(frame,bitmask=None,ivar_framemask=True):
+    """
+        Wrapper script of get_fiberbitmasked_frame_arrays that will
+        return a modified version of the cframe instead of just the 
+        flux and ivar
+    """
+    ivar,mask = get_fiberbitmasked_frame_arrays(frame,bitmask,ivar_framemask,return_mask=True)
+    outframe = frame
```

Comment:
Okay I'll make that change. It initially read:   outframe = frame.copy()
but that throws an error since frame doesn't have a copy helper function. Rather than deleting the line I lazily just removed the .copy().

I'll change it to just frame and make it explicit in the heading of the function.
I think the copy is probably unnecessary given we're reassigning back to "frame" in all locations this function is used in the code anyway.